tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_ms.py

```diff
@@ -55,25 +60,6 @@ def get_node_pod_names_expected_to_terminate(node_name):
     ]
 
 
-def get_all_pod_names_expected_to_terminate():
```

Comment:
Are you sure we'll never need this function for any disaster recovery test cases?